examples/api/lib/cupertino/date_picker/cupertino_date_picker.0.dart

```diff
@@ -0,0 +1,190 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This sample shows how to implement CupertinoDatePicker with different picker modes.
```

Comment:
This is a great description, but it should probably be the description in the API doc comment in the original code.

Sorry, I should have mentioned this earlier, but when building the API docs site, this comment will get stripped from the example, because we strip comments up to the first non-comment in order to eliminate the copyright from the display in the API docs, so this comment won't appear in the published API docs.